tests/test_series.py

```diff
@@ -416,10 +416,11 @@ def test_types_sort_values_with_key() -> None:
 
 
 def test_types_shift() -> None:
-    s = pd.Series([1, 2, 3])
+    s = pd.Series([1, 2, 3], index=pd.date_range("2020", periods=3))
```

Comment:
🤔 trying to add a test for this, the type actually changes from Series['int'] to Series['float'], taking a look